tests/sb-util.test.ts

```diff
@@ -162,29 +162,88 @@ describe('Sprite class -------------------------', () => {
+            expect(globalListVar.global()).toBeTruthy();
+        });
     });
 
-    test('get all variables on a Sprite and make sure global scope vars from stage included', () => {
-        const variables = spriteWithVars.variables();
+    describe('variable tests on sprite', () => {
+        let variables;
 
-        const localVar = variables.byId('58`Aw=nriuw[2/#s@20!-X vel');
-        expect(localVar.prop('name')).toEqual('X vel');
-        expect(localVar.prop('value')).toEqual(-0.44432764720836493);
-        expect(localVar.global()).toBeFalsy();
+        beforeAll(() => {
+            variables = spriteWithVars.variables();
+        });
 
-        // variable in global scope
-        const globalVar = variables.byId('1B!G;|;P6ALif|?9(Tum-glitch_Acceleration');
-        expect(globalVar.prop('name')).toEqual('glitch_Acceleration');
-        expect(globalVar.prop('value')).toEqual('50');
+        test('get a local scalar variable on a Sprite', () => {
+            const localScalarVar = variables.byId('58`Aw=nriuw[2/#s@20!-X vel');
+            expect(localScalarVar.prop('name')).toEqual('X vel');
+            expect(localScalarVar.prop('value')).toEqual(-0.44432764720836493);
+            expect(localScalarVar.global()).toBeFalsy();
+        });
+
+        test('get a local list variable on a Sprite', () => {
+            const localListVar = variables.byId('2Q_=y9u5Qq|~fG25;O7V');
+            expect(localListVar.prop('name')).toEqual('sprite-list');
+            expect(
+                localListVar
+                    .prop('value')
+                    .map((v: string) => +v)
+                    .reduce((a: number, b: number) => a + b),
+            ).toEqual(66);
+        });
+
+        test('global scalar var from stage available', () => {
+            // scalar variable in global scope
+            const globalScalarVar = variables.byId('1B!G;|;P6ALif|?9(Tum-glitch_Acceleration');
+            expect(globalScalarVar.prop('name')).toEqual('glitch_Acceleration');
+            expect(globalScalarVar.prop('value')).toEqual('50');
+            expect(globalScalarVar.global()).toBeTruthy();
+        });
+
+        test('global broadcast var from stage available', () => {
+            // broadcast variable in global scope
+            const globalBroadcastVar = variables.byId('broadcastMsgId-start glowing (accelerator)');
+            expect(globalBroadcastVar.prop('name')).toEqual('start glowing (accelerator)');
+            expect(globalBroadcastVar.prop('value')).toEqual('start glowing (accelerator)');
+            expect(globalBroadcastVar.global()).toBeTruthy();
+        });
+
+        test('global list var from stage available', () => {
+            const globalListVar = variables.byId('Pi3E!fcsKWLQ}5(ugkcS');
+            expect(globalListVar.prop('name')).toEqual('global-list');
+            expect(
+                globalListVar
+                    .prop('value')
+                    .map((v: string) => +v)
```

Comment:
rather than `+v` can you just use `.map(Number)` ?

`+v` is an old trick that definitely works, but `Number(v)`  or `parseFloat(v)` are much more readable / intentions are known

I generally like `Number(v)` cuz it also works for `Boolean(v)` instead of `!!v` and `String(v)` if you really want to make sure it's a string.